newsfragments/544.feature

```diff
@@ -0,0 +1 @@
+``tox --version`` now shows information about all registered plugins - by :user:`obestwalter`
```

Comment:
Interesting, does this `by :user:`obestwalter` trick work?

Reply:
yes it does 👍

Reply:
Interesting, didn't find this in the docs, I will take a look at the code when I have the time to figure out what other tricks are hidden there.